docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -803,19 +803,26 @@ in the following sections:
 
 === Spring Cloud LoadBalancer Caching
 
-Apart from the basic `ServiceInstanceListSupplier` implementation that retrieves instances
-via `DiscoveryClient` each time it has to choose an instance, we provide a https://github.com/ben-manes/caffeine[Caffeine-backed]
-implementation.
-To make use of it, you need to have `com.github.ben-manes.caffeine:caffeine` in the classpath.
+Apart from the basic `ServiceInstanceListSupplier` implementation that retrieves instances via `DiscoveryClient` each time it has to choose an instance, we provide two caching implementations:
```

Comment:
I think we need to organize this caching documentation a little better.

Can we break things into 3 sections

1. No caching library on the classpath
2. Caffeine on the classpath
3. Evictor on the classpath